src/js/common/schemaform/widgets/index.js

```diff
@@ -3,13 +3,17 @@ import SelectWidget from './SelectWidget';
 import DateWidget from './DateWidget';
 import EmailWidget from './EmailWidget';
 import RadioWidget from './RadioWidget';
+import CheckboxWidget from './CheckboxWidget';
+import YesNoWidget from './YesNoWidget';
 
 const widgets = {
   TextWidget,
   SelectWidget,
   DateWidget,
   EmailWidget,
-  RadioWidget
+  RadioWidget,
+  CheckboxWidget,
+  yesNo: YesNoWidget
```

Comment:
Similar comment with the review version of it; why not just have `widgets.YesNoWidget` for consistency?